<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>data_entry</class>
 <widget class="QDialog" name="data_entry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>680</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveaddButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>610</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save and add another</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="contactBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Contact Type</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="diagnosisBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>231</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>Diagnosis</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="investigationBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>231</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Investigations</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="referralBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>221</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Referrals</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="disposalBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Disposal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>300</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Follow up time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="followupEdit">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>320</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="admitCheckBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>230</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Admit from clinic</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="radiologyCheckBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>260</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>For radiology meeting</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="seenbyCheckBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>600</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Seen by other</string>
   </property>
  </widget>
  <widget class="QComboBox" name="seenbyCombo">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>630</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="enteredLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="notesEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>211</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="diagEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>570</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ixEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="refEdit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>520</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>201</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>NHS</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>RXR</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nhsEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="rxrEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>550</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear form</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>data_entry</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>data_entry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
